openedx/features/personalized_learner_schedules/call_to_action.py

```diff
@@ -81,6 +81,17 @@ def _is_block_shiftable(xblock):
             PersonalizedLearnerScheduleCallToAction._log_past_due_warning(type(xblock).__name__)
             is_past_due = xblock.is_past_due
 
+        # This is used to replicate the logic used in common/lib/xmodule/xmodule/vertical_block.py
```

Comment:
Without looking at the code myself, this makes me wonder if we can't consolidate logic? But I'm guessing you already considered that and we can't easily.

Reply:
Hmm. I might be able to write a method on the CAPA xblock itself that contains this. I wonder if that might miss some blocks though (such as LTI). Maybe it doesn't matter?